combined_browser_metrics/views/active_users_aggregates.view.lkml

```diff
@@ -270,6 +270,26 @@ view: +active_users_aggregates {
     sql:  ${TABLE}.active_hours ;;
   }
 
+  dimension: is_default_browser {
```

Comment:
LGTM. Would be great if @Marlene-M-Hirose @ANich can confirm that changing the type from boolean to string in the view doesn't affect your querying in Looker.